server/scripts/create_database.sql

```diff
@@ -21,8 +21,39 @@ CREATE TABLE clients
     language_spoken VARCHAR(120),
     english_speaker BOOLEAN,
     english_skill_level VARCHAR(120),
-    gender INT,
+    gender VARCHAR(120),
     date_of_birth DATE,
     postcode VARCHAR(20),
     referring_agency VARCHAR(120)
 );
+
+CREATE TABLE collections
```

Comment:
It looks like we've added a collections table which is part of another ticket—can we remove this?